spec/lib/db_seed_spec.rb

```diff
@@ -1,6 +1,8 @@
 require "rails_helper"
 
 RSpec.describe "db:seed" do
+  setup { Casa::Application.load_tasks }
```

Comment:
This should be `before`. `setup` is really a Rails test method, but rspec converts it to `before`.